src/core/IconButton/index.stories.tsx

```diff
@@ -28,72 +29,158 @@ export default {
 
 const Template: Story = (args) => <Demo {...args} />;
 
-export const LargePrimaryIconButton = Template.bind({});
+export const Default = Template.bind({});
 
-LargePrimaryIconButton.args = {
+Default.args = {
   "aria-label": "info",
   disabled: false,
   icon: <Icon sdsIcon="infoCircle" sdsSize="xl" sdsType="iconButton" />,
   sdsSize: "large",
   sdsType: "primary",
 };
 
-export const LargeSecondaryIconButton = Template.bind({});
-
-LargeSecondaryIconButton.args = {
-  "aria-label": "checked",
-  disabled: false,
-  icon: <Icon sdsIcon="checkCircle" sdsSize="xl" sdsType="iconButton" />,
-  sdsSize: "large",
-  sdsType: "secondary",
+Default.parameters = {
+  snapshot: {
+    skip: true,
+  },
 };
 
-export const LargeTertiaryIconButton = Template.bind({});
+export const Test = Template.bind({});
 
-LargeTertiaryIconButton.args = {
-  "aria-label": "x mark",
+Test.args = {
+  "aria-label": "info",
   disabled: false,
-  icon: <Icon sdsIcon="xMark" sdsSize="xl" sdsType="iconButton" />,
+  icon: <Icon sdsIcon="search" sdsSize="xl" sdsType="iconButton" />,
   sdsSize: "large",
-  sdsType: "tertiary",
-};
-
-export const MediumTertiaryIconButton = Template.bind({});
-
-MediumTertiaryIconButton.args = {
-  "aria-label": "x mark",
-  disabled: false,
-  icon: <Icon sdsIcon="xMark" sdsSize="l" sdsType="iconButton" />,
-  sdsSize: "medium",
-  sdsType: "tertiary",
+  sdsType: "secondary",
 };
 
-export const SmallPrimaryIconButton = Template.bind({});
+const LivePreviewDemo = (props: Args): JSX.Element => {
+  const spacings = getSpaces(props);
 
-SmallPrimaryIconButton.args = {
-  "aria-label": "info",
-  disabled: false,
-  icon: <Icon sdsIcon="infoCircle" sdsSize="s" sdsType="iconButton" />,
-  sdsSize: "small",
-  sdsType: "primary",
-};
-
-export const SmallSecondaryIconButton = Template.bind({});
+  const livePreviewStyles = {
+    alignItems: "center",
+    display: "grid",
+    gridColumnGap: "0px",
+    gridRowGap: "50px",
+    gridTemplateColumns: "repeat(11, min-content)",
+  };
 
-SmallSecondaryIconButton.args = {
-  "aria-label": "checked",
-  disabled: false,
-  icon: <Icon sdsIcon="checkCircle" sdsSize="s" sdsType="iconButton" />,
-  sdsSize: "small",
-  sdsType: "secondary",
+  return (
+    <div style={livePreviewStyles as React.CSSProperties}>
+      <div>
```

Comment:
Merge these divs so that the 24px gap between the columns would be ok.

```
<div style={{ display: "flex" }}>
  <Demo
    style={{ marginRight: spacings?.xxs }}
    icon={<Icon sdsIcon="grid" sdsSize="l" sdsType="iconButton" />}
    sdsSize="large"
    sdsType="primary"
  />

  <Demo
    style={{ marginRight: spacings?.xxs }}
    icon={<Icon sdsIcon="grid" sdsSize="l" sdsType="iconButton" />}
    sdsSize="large"
    sdsType="primary"
  />
</div>
```